prelexer.cpp

```diff
@@ -423,6 +423,9 @@ namespace Sass {
     const char* number(const char* src) {
       return sequence< optional<sign>, unsigned_number>(src);
     }
+    const char* sass_number(const char* src) {
+      return sequence< optional<sign>, unsigned_number, optional < identifier > >(src);
```

Comment:
I wont get a chance to look at this anytime soon. If this fixes an issue :+1: I'll open another issue to take a look at static value parsing.